### PR TITLE
Add manifest

### PR DESCRIPTION
**Pull request by DaMan69**
This tells Windows the program supports 7/8/10, is DPI aware, and enables long filename support on W10.

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/2911)

### CODE REVIEW

File: src/citra/citra.rc [hunk 1]
```diff
@@ -6,4 +6,4 @@
 // Icon with lowest ID value placed first to ensure application icon
 // remains consistent on all systems.
 CITRA_ICON              ICON                    "../../dist/citra.ico"
-
+1                       24                      "../../dist/citra.manifest"
```
Comment (Subv) [inline src/citra/citra.rc]: Aren't there any constants for this?

Comment (jroweboy) [reply src/citra/citra.rc]: they are defined in a different header so you'd need to include it (`WinUser.h`) but the named constant is `RT_MANIFEST`

**Comment by DaMan69** [comment]
Updated with Subv's feedback.

**Comment by yuriks** [comment]
Is disabling the linker manifest and the manual embedding necessary? My understanding is that you can give a manifest to the build system, and it'll merge it with the linker-generated one. CMake supports this by [supplying the manifest file as a source file](https://cmake.org/cmake/help/v3.4/release/3.4.html?highlight=manifest#other).

**Comment by DaMan69** [comment]
You are correct for MSVC, but it is necessary for MinGW.

**Pull request merged**